std/format/spec.d

```diff
@@ -806,8 +806,8 @@ Params:
 Returns:
     A `FormatSpec` with the specifier parsed.
 Throws:
-    A `FormatException` when more than one specifier is given or the specifier
-    is malformed.
+    A `FormatException` when more than one specifier is given, the specifier
+    is malformed, or when the specifier is "%%".
```

Comment:
```suggestion
    is malformed or when the specifier is "%%".
```

Reply:
With #7891 being merged, this isn't relevant anymore. There are more cases, where a `FormatException` can be thrown, so I used a more general term there and there is no need any more to write anything about `%%`.

But: When handling the merge conflicts, the "Known Bug" section should be removed.

Reply:
Thanks for the heads-up.